### PR TITLE
Avatar Component

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
Added -
profile Avatar
Size Avatar

### CODE REVIEW

File: components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }
+  
+  .main--content h2 {
+    font-size: 2rem;
+    font-weight: 900;
+  }
+  
+  .main--content p {
+    font-size: 1.7rem;
+    padding: 2rem 0 1rem 0;
```
Comment (rkapoor10) [inline components/Avatar/avatar.css:14]: give these element tags in base css

File: components/Avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+            <div class="divider"></div>
+
+            <h2>Size Avatar </h2>
+          <p>This can be created by using <code>avatar</code> class and size class like   <code> sm | md | lg </code> in an <code>img</code> tag. </p>
+  <div class="showcase">
+            <img class="avatar sm" src="/components/Avatar/images/man (1).png" alt="" srcset="">
+            <img class="avatar md" src="/components/Avatar/images/man.png" alt="" srcset="">
+            <img class="avatar lg" src="/components/Avatar/images/gamer.png" alt="" srcset="">
+  </div>
```
Comment (rkapoor10) [inline components/Avatar/avatar.html:100]: remove srcset if not in use

**Review by rkapoor10 — APPROVED** [review]
Good to merge.

File: components/Avatar/avatar.html (hunk 2 above)
Comment (SuyashTiwariCoder) [reply components/Avatar/avatar.html:100]: ok

**Review by Raghav888 — APPROVED** [review]

**Pull request merged**